src/pages/monetization/how-to-monetize-apps/index.mdx

```diff
@@ -153,9 +153,16 @@ You can check your current account balance in the **My Revenue** section. You'll
 
 <img
   alt="LiveChat My Revenue Section"
-  src="/images/monetization/how-to-monetize-apps/livechat-my-revenue.png"
-  width="800px"
-  height="590px"
+  src="/images/monetization/how-to-monetize-apps/livechat-monetization-revenue.png"
+  width="1320px"
+  height="897px"
+/>
+
+<img
+  alt="LiveChat My Revenue Section"
+  src="/images/monetization/how-to-monetize-apps/livechat-monetization-revenue1.png"
+  width="1130px"
```

Comment:
Is this the correct size? The image seems a little squeezed https://deploy-preview-1139--livechat-public-docs.netlify.app/monetization/how-to-monetize-apps

Reply:
You are totally right, I looked too much at all of them and did not notice it. Already changed, should be fine now.